data/tabular/opv/meta.yaml

```diff
@@ -0,0 +1,165 @@
+---
+name: opv
+description: |-
+    Database of organic nonfullerene solar cells with their composition, device stacks, and performance.
+targets:
+    - id: PCE_max(%)
+      description: Maximum power conversion efficiency of devices with this polymer
+      units: '%'
+      type: continuous
+      names:
+          - noun: maximum power conversion efficiency of tested devices
+          - noun: maximum achievable power conversion efficiency
+      uris:
+      significant_digits: 2
+      sample: false
+    - id: PCE_ave
+      description: Average power conversion efficiency of devices with this polymer
+      units: '%'
+      names:
+          - noun: power conversion efficiency (PCE)
+          - noun: power conversion efficiency
+      type: continuous
+      significant_digits: 2
+    - id: Voc
+      description: Open-circuit voltage of devices with this polymer
+      units: V
+      type: continuous
+      names:
+          - noun: open-circuit voltage of tested devices
+          - noun: open-circuit voltage
+      uris:
+      significant_digits: 2
+    - id: Jsc
+      description: Short-circuit current density of devices with this polymer
+      units: mA/cm^2
+      type: continuous
+      names:
+          - noun: short-circuit current density of tested devices
+          - noun: short-circuit current density
+      uris:
+      significant_digits: 2
+    - id: FF
+      description: Fill factor of devices with this polymer
+      units: ''
```

Comment:
dimensionless?

Reply:
If, I would exclude it from the general templates, as they always assume a unit.